index.bs

```diff
@@ -14601,19 +14608,28 @@ requirements beyond the normal ones for [=interface types=].
 Each {{DOMException}} object has an associated <dfn for="DOMException">name</dfn> and
 <dfn for="DOMException">message</dfn>, both [=strings=].
 
+Each {{DOMException}} object has an associated <dfn for="DOMException">cause</dfn>, which
+is a JavaScript value. It is {{undefined}} unless specified otherwise.
+
 The
-<dfn constructor for="DOMException" lt="DOMException(message, name)"><code>new DOMException(|message|, |name|)</code></dfn>
+<dfn constructor for="DOMException" lt="DOMException(message, options)"><code>new DOMException(|message|, |options|)</code></dfn>
 constructor steps are:
 
-1. Set [=this=]'s [=DOMException/name=] to |name|.
-1. Set [=this=]'s [=DOMException/message=] to |message|.
+1.  Set [=this=]'s [=DOMException/message=] to |message|.
+1.  If |options| is a string, then set [=this=]'s [=DOMException/name=] to |options|.
+1.  Otherwise,
+    1.  Set [=this=]'s [=DOMException/name=] to |options|["{{DOMExceptionOptions/name}}"].
+    1.  Set [=this=]'s [=DOMException/cause=] to |options|["{{DOMExceptionOptions/cause}}"].
```

Comment:
You have to handle the case where `|options|["{{DOMExceptionOptions/cause}}"]` does not `[=map/exist=]`.

A proposal to make this easier is https://github.com/whatwg/webidl/issues/983 but it is not available yet.

Reply:
Thanks, fixed.